src/app/technical-record/vehicle-summary/vehicle-summary.component.html

```diff
@@ -21,7 +21,6 @@
       </dd>
       <dd class="govuk-summary-list__value"></dd>
     </div>
-
```

Comment:
For this file, lets take the version from the main 10202 branch. Seems the line separator are removed

Reply:
added back